test/e2e/app/app_test.go

```diff
@@ -113,6 +113,16 @@ var _ = ginkgo.Describe("App", func() {
 			gomega.Ω(appCRD.Spec.TaskGroups[0].Tolerations[0].Effect).To(gomega.Equal(v1.TaintEffectNoSchedule))
 		})
 
+		ginkgo.It("Verify the Yunikorn Scheduler healthy", func() {
+			ginkgo.By("Call the HealthCheck API")
+			healthCheck, err := yunikorn.GetHealthCheck()
+			gomega.Ω(err).NotTo(gomega.HaveOccurred())
+			gomega.Ω(healthCheck.Healthy).To(gomega.BeTrue())
+			for _, check := range healthCheck.HealthChecks {
+				gomega.Ω(check.Succeeded).To(gomega.BeTrue())
+			}
```

Comment:
if line 120 returns true, I do not think we need to double-check every check.
If any check fails, the overall result should be false.
however, one thing not very clear to me is, when this fails, can we print the detailed messages in the log to help us debug?

Reply:
On failed assertions, gomega can also print a formatted log: https://onsi.github.io/gomega/#annotating-assertions